docs/projects/hello-world/_posts/2018-11-17-red.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Hello World in Red
+layout: default
+last-modified: 2018-11-17
+featured-image:
+tags: [red, hello-world]
+authors:
+  - the_renegade_coder
+---
+## How to Implement the Solution
+In this article, we'll demonstrate how to implement Hello World in Red.
```

Comment:
Looks good! Perhaps I should have been more clear: the intro paragraph should go above the heading.

```
In this article, we'll demonstrate how to implement Hello World in Red.

## How to Implement the Solution
```

Reply:
I fixed it real quick!